test/integration_tests.py

```diff
@@ -1,10 +1,26 @@
 import asyncio
 import asynqp
 import socket
+import urllib.request
+import base64
 import contexts
+import json
 from .util import testing_exception_handler
 
 
+def _deep_compare(a, b):
```

Comment:
Apparently, you are right. I was thinking, that `==` would also take the order of the `dict` into account.